api/bin/create_erds.py

```diff
@@ -1,12 +1,9 @@
 # Generate database schema diagrams from our SQLAlchemy models
-import codecs
 import logging
-import os
 import pathlib
 from typing import Any
 
-import pydot
-import sadisplay
+from sqlalchemy_schemadisplay import create_uml_graph
```

Comment:
I'm not sure I like `sqlalchemy_schemadisplay` - it changes quite a bit in our ERDs that just makes them incorrect. It doesn't look like it picks up foreign keys for a large percentage of our tables, and our non-foreign key relationships that we wouldn't want in an ERD (they're not actually in the DB, just an ORM-level thing).

Did you get a chance to test out eralchemy? From my 5 minutes of testing it seemed a bit closer to what we currently have with pretty minimal configuration (just give it the metadata object)